docs/scraping-receivers.md

```diff
@@ -0,0 +1,143 @@
+
+There are two categories of the metrics emitted by scraping receivers:
+
+- **Default metrics**: emitted by default, but can be disabled in user settings.
+
+- **Optional metrics**: not emitted by default, but can be enabled in user settings.
+
+### How to identify if new metric should be default or optional?
+
+There is no strict rule to differentiate default metrics from optional. As a rule of thumb, default metrics SHOULD be 
+treated as metrics essential to determine healthiness of a particular monitored entity. Optional metrics on the 
+other hand SHOULD be treated as a source of additional information about the monitored entity. 
+
+If two metrics are built based on the same data but have different representation, at least one of them MUST be 
+optional. For example:
+
+- `system.cpu.time` (CPU time reported as cumulative sum in seconds) - Default metric.  
+- `system.cpu.utilization` (fraction of CPU time spent in different states reported as gauge) - Optional metric.
+
+In this example, system CPU usage time can be emitted as two different metrics: a gauge as a fraction of 1 or 
+a cumulative sum in seconds. Potentially, each of them can be deducted from one another, that's why only one of 
+them is enabled by default.
+
+## Stability levels of scraping receivers
+
+All the requirements defined for components in [Collector README](../README.md#stability-levels) are applicable to 
+the scraping receivers as well. In addition, the following rules applied specifically to scraping metrics receivers:
+
+### In development
+
+The receiver is not ready for use. All the metrics emitted by the receiver are not finalized and can change in any way.
+
+### Alpha
+
+The receiver is ready for limited non-critical workloads. The list of emitted default metrics SHOULD be 
+considered as complete, but any changes to the `metadata.yaml` still MAY be applied.
+
+### Beta
+
+The receiver is ready for non-critical production workloads. The list of emitted default metrics MUST be
+considered as complete. Breaking changes to the emitted metrics SHOULD be applied following [the deprecation 
+process](#changing-the-emitted-metrics). All the metric and resource attribute names SHOULD be released in an 
+[OpenTelemetry Specification](https://github.com/open-telemetry/opentelemetry-specification).
+
+### Stable
+
+The receiver is ready for production workloads. Breaking changes to the emitted metrics SHOULD be avoided. If 
+a change is required, it MUST be applied following [the deprecation process](#changing-the-emitted-metrics). All 
+the metric and resource attribute names MUST be released in an [OpenTelemetry 
+Specification](https://github.com/open-telemetry/opentelemetry-specification)
+
+## Changing the emitted metrics
+
+Some changes are not considered breaking and can be applied to metrics emitted by scraping receivers of any 
+stability level:
+
+- Adding a new optional metric.
+- Adding a new resource attribute to existing metrics.
```

Comment:
Some backends would consider this breaking because metrics would become associated with a new resource.

Reply:
I'll remove this line for now. But potentially we can consider an ability to hide non-identifying resource attribute in user settings. In that case it'll become:
```
- Adding a new optional resource attribute.
```

Reply:
That would be nice. In general the lack of distinction between identifying and non-identifying resource attributes is problematic.